app/routes/public/sessions/list.js

```diff
@@ -176,6 +176,10 @@ export default Route.extend({
         ]
       });
     }
+    // sorting the sessions according to their average rating
+    sessions.content.sort((a, b) => {
```

Comment:
I suggest you read more about Ember.js. 
Querying/Fetching the store gives us many in hand options rather than using the native sorting.

Reply:
working on it sir

Reply:
For reference you may see other files like in `routes/index.js` we have sorted events on basis of start time. You will get an idea about it!

Reply:
I think there is a problem with the server while responding to get requests of the form 
https://open-event-api-dev.herokuapp.com/v1/sessions?sort=average-rating&filter=[]

you can try the following two get requests to identify
https://open-event-api-dev.herokuapp.com/v1/sessions?sort=created-at
https://open-event-api-dev.herokuapp.com/v1/sessions?sort=average-rating

While the first one is a success, the second is failure. The requests is success for all other params which I have tried. How should I proceed now?